pkg/specgen/generate/security.go

```diff
@@ -144,11 +138,22 @@ func securityConfigureGenerator(s *specgen.SpecGenerator, g *generate.Generator,
 		configSpec.Process.Capabilities.Inheritable = caplist
 		configSpec.Process.Capabilities.Ambient = caplist
```

Comment:
not related to the patch, but why do we set Ambient capabilities for root containers?

Reply:
I am not quite sure how to handle these different caps, But logging in as root, I see that capamb and capinh by default are all 0's so we probably should not touch them in either case.